.gitignore

```diff
@@ -2,3 +2,4 @@
 /telegraf
 /telegraf.exe
 /telegraf.gz
+vendor/
```

Comment:
Nitpick: `/vendor` because what if I want to make `internal/vendor/stupid_example.go`?